client/src/components/Instructions/index.js

```diff
@@ -1,103 +1,102 @@
 import React, { Component } from "react";
 import { Button } from "rimble-ui";
-import styles from './Instructions.module.scss';
+import styles from "./Instructions.module.scss";
 
 export default class Instructions extends Component {
-
   renderCounterSetup() {
     const addressDefault = this.getDefaultAddress();
     return (
       <div className={styles.instructions}>
         <h2> Build your first app with ZepKit </h2>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Add the Counter contract to your project.
+            Initialize your ZeppelinOS project
           </div>
           <div className={styles.code}>
-            <code>
-              zos add Counter
-            </code>
+            <code>zos init {"<<your project name>>"}</code>
           </div>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Connect with your local blockchain by opening a session (Note that we automatically prefilled your ganache account).
+            Add the Counter contract to your project.
           </div>
           <div className={styles.code}>
-            <code>
-              zos session --network development --from {addressDefault} --expires 3600
-            </code>
+            <code>zos add Counter</code>
           </div>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Let's compile the Counter contract.
+            Let's compile the Counter contract, follow the <span className={styles.inline}>cli</span>  prompts.
           </div>
           <div className={styles.code}>
-            <code>
-              zos push --deploy-dependencies
-            </code>
+            <code>zos push</code>
           </div>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            We create an instance of our contract and deploy it.
+            We create an instance of our contract and deploy it, follow the <span className={styles.inline}>cli</span> 
+            prompts.
           </div>
           <div className={styles.code}>
-            <code>
-              zos create Counter --init initialize --args 2
-            </code>
+            <code>zos create Counter</code>
           </div>
         </div>
         <div className={styles.step}>
           <div className={styles.instruction}>
-            Add a log to verify that the contract is available. Before line 77 in App.js, add:
+            Add a log to verify that the contract is available. Before line 77
```

Comment:
What is the purpose of this step? Line number may change at any time. It is error prone. I suggest we either put `console.log` ourselves there or remove it all together?

Reply:
This was in the project originally. Honestly, I think it's just because it's somewhat unreliable if the contract instance is created. Mostly, in my opinion, because Metamask doesn't always properly refresh, or because you're on the wrong network. I think Ramon added it so you could do a sanity check to figure out why it's not working. I frequently do the same because I have the same problem. 

That said, I agree it is error prone and a bit strange as a suggestion. Breaks our "easy to use" promise to the user I think.

Reply:
Okay. Let's just put console.log(Contract) ourselves there. If it breaks there will be an error in browser output.